packs/menace-under-otari-bestiary/zombie-shambler-bb.json

```diff
@@ -88,11 +88,46 @@
             },
             "type": "melee"
         },
+        {
+            "_id": "SyWmilYaLnrtDFVe",
+            "flags": {
+                "core": {
+                    "sourceId": "Compendium.pf2e.bestiary-ability-glossary-srd.Item.qCCLZhnp2HhP3Ex6"
+                }
+            },
+            "img": "systems/pf2e/icons/actions/Passive.webp",
+            "name": "Darkvision",
```

Comment:
We actually removed all these LLV and darkvision passives from NPCs a few weeks ago. I can just take them back out in a later update though since it was a script that did it.